test/e2e/test.js

```diff
@@ -0,0 +1,231 @@
+                `${rootPath}/po:/insight/node_modules/@dashevo/insight-ui/po`,
+                `${rootPath}/public:/insight/node_modules/@dashevo/insight-ui/public`,
+            ],
+        };
+
+        const insighUIOptions = {
+            cacheNodeModules: true,
+            localAppPath: rootPath,
+            container: insightUIContainerOptions,
+        };
+
+        InsightUIOptions.setDefaultCustomOptions(insighUIOptions);
+
+
+        [masterNode] = await startInsightUI.many(1);
+
+        url = `http://127.0.0.1:${masterNode.insightUi.options.getUiPort()}/insight/`;
+
+        await masterNode.dashCore.getApi().generate(15);
+
+    });
+
+    afterAll(async () => {
+        const instances = [
+            masterNode,
+        ];
+
+        await Promise.all(instances.filter(i => i)
+            .map(i => i.remove()));
+
+    });
+
+    beforeEach(function () {
+        originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = 250000;
+    });
+
+    afterEach(function () {
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout;
+    });
+
+    describe('Home Page', () => {
+        it('should be able to open main page', async () => {
+            await browser.get(url);
+            const title = await browser.getTitle();
+            expect(title).equal('Home | Insight');
+        });
+
+        it('should be able to open block page', async () => {
+            await topPanel.openBlockPage();
+            const title = await browser.getTitle();
+            expect(title).equal('Home | Insight');
+        });
+
+        it('should be able to open status page', async () => {
+            await topPanel.openStatusPage();
+            await wait(10000); // time to complete sync
+            const title = await browser.getTitle();
+            expect(title).equal('Status | Insight');
+
+            const syncProgress = await statusPage.getSyncProgress();
+            expect(syncProgress).equal("100% Complete");
+
+            const currentSyncStatus = await statusPage.getCurrentSyncStatus();
+            expect(currentSyncStatus).equal("finished");
+
+            const startDate = await statusPage.getStartDate();
+            expect(startDate).equal("Invalid date");
+
+            const initialBlockChainHeight = await statusPage.getInitialBlockChainHeight();
+            expect(Number.isInteger(parseInt(initialBlockChainHeight))).equal(true);
+
+            const syncedBlocks = await statusPage.getSyncedBlocks();
+            expect(syncedBlocks).equal("");
+
+            const skippedBlocks = await statusPage.getSkippedBlocks();
+            expect(skippedBlocks).equal("");
+
+            const syncType = await statusPage.getSyncType();
+            expect(syncType).equal("bitcore node");
+
+            const lastBlockHash = await statusPage.getLastBlockHash();
+            expect(lastBlockHash).not.equal(undefined);
```

Comment:
expect.to.not.be.undefined

Reply:
the same 
  Message:
    Failed: Cannot read property 'equal' of undefined
  Stack:
    TypeError: Cannot read property 'equal' of undefined
        at UserContext.it (/Users/andrei/dash_src/insight-ui/test/e2e/test.js:87:23)